qiskit_nature/drivers/second_quantization/pyscfd/pyscfdriver.py

```diff
@@ -540,11 +542,9 @@ def _construct_driver_result(self) -> ElectronicStructureDriverResult:
         self._populate_driver_result_electronic_energy(driver_result)
         self._populate_driver_result_electronic_dipole_moment(driver_result)
 
-        # TODO: once https://github.com/Qiskit/qiskit-terra/issues/6772 is resolved, we no longer
-        # _have_ to add these properties. However, until then the interpret method relies on indices
-        # of the aux_operators which are incorrect if these properties are not added.
-        driver_result.add_property(AngularMomentum(self._mol.nao * 2))
-        driver_result.add_property(Magnetization(self._mol.nao * 2))
+        if not settings.dict_aux_operators:
```

Comment:
Yes this will need to be updated. However, to my understanding, Qiskit Terra updated the deprecation policy stating that a deprecated feature can only be removed if:
- it has been deprecated for at least 3 months
- AND at least one minor feature release has occurred between the deprecation and removal

This is to avoid that e.g. something gets deprecated in 0.3 (like here) and immediately removed in 0.4, giving users time to adjust. Thus, we will only be able to fully switch to dict-based aux operators in 0.5

Reply:
It seems I had not seen part 3a here https://qiskit.org/documentation/contributing_to_qiskit.html#deprecation-policy Given the usual release interval we have deprecated logic is normally going to removed 2 versions later then. So far we have never released a minor version update within 3 months of the prior version. Ok, so the above stays for a while more yet then!!